pkgs/build-support/rust/default.nix

```diff
@@ -173,29 +173,31 @@ stdenv.mkDerivation (filteredArgs // {
         --frozen ${concatStringsSep " " cargoBuildFlags}
     )
 
-    # rename the output dir to a architecture independent one
-    mapfile -t targets < <(find "$NIX_BUILD_TOP" -type d | grep '${releaseDir}$')
-    for target in "''${targets[@]}"; do
-      rm -rf "$target/../../${buildType}"
-      ln -srf "$target" "$target/../../"
-    done
```

Comment:
Why was moving this block necessary?

Reply:
Ah in order to make cargo test remove the code.

Reply:
That's right. I guess this can be resolved then? : )